packages/quantic/force-app/main/default/labels/CustomLabels.labels-meta.xml

```diff
@@ -1442,4 +1442,46 @@
     <protected>false</protected>
     <shortDescription>Show less</shortDescription>
   </labels>
+  <labels>
+    <fullName>quantic_SearchFieldWithSuggestions</fullName>
+    <value>Search field with suggestions. Suggestions may be available under this field. To send, press Enter.</value>
```

Comment:
Question here. This label reads a bit odd if not in a aria context. Do we want to create "aria-specific" labels? Meaning labels only meant for screen readers/accessibility purposes or we want to keep everything uniform? I was thinking we could maybe name the labels for aria purposes something like: `quantic_Aria_SearchFieldWithSuggestions`

Reply:
While it can be a nice idea to do that I don't necessarily see the value it will add versus the naming rule that will be added up.